src/test/java/org/opensrp/repository/postgres/LocationRepositoryTest.java

```diff
@@ -254,7 +255,7 @@ public void testUpdateLocation() throws ParseException {
 		
 		assertNotNull(updatedLocation);
 		assertEquals(GeometryType.POLYGON, updatedLocation.getGeometry().getType());
-		assertEquals(PropertyStatus.PENDING_REVIEW, updatedLocation.getProperties().getStatus());
+		assertEquals(PropertyStatus.ACTIVE, updatedLocation.getProperties().getStatus());
```

Comment:
Why is this modified

Reply:
The `get({identifier})` method has been updated to return the `active` location.
This section of the test has also been updated https://github.com/OpenSRP/opensrp-server-core/pull/268/files#diff-185cc4dc43c21f1f9757f284ef3621afR245